modules/aks/starterkit/buildingblock/outputs.tf

```diff
@@ -25,9 +25,9 @@ This starter kit has set up the following resources in workspace `${var.workspac
 
 - **GitHub Repository**: [${local.identifier}](<${data.meshstack_building_block_v2.repo_data.status.outputs.repo_html_url.value_string}>)
```

Comment:
d: I think this link is wrong, ie im not sure the panel loads with the platform tenant id 

btw: @sttomm has added a new mention syntax support to meshStack that we can already leverage here (but this can also just be another pr tbh) which would remove that type of issue entirely